packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html

```diff
@@ -36,16 +36,6 @@
     label="{{:: 'hosting_multisite_domain_configuration_options_choose' | translate }}"
     size="l"
 >
-    <oui-checkbox name="ipv6" model="$ctrl.ipv6">
-        <span
-            data-translate="hosting_multisite_domain_configuration_ipv6"
```

Comment:
If both translation keys are no more used, then you can also removed them from:
- `packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json`.

Reply:
i remember that we discuss about a tool which list all no used tran. keys, right ?
Can we use it to remove keys or not ?

Reply:
also how can we be sure the key is not used in split mode ?
Like: 'hosting_tab_DOMAINS_configuration_add_step2_mode_' + selected.mode + '_information'

Reply:
I removed all non used keys

Reply:
> i remember that we discuss about a tool which list all no used tran. keys, right ?
> Can we use it to remove keys or not ?

Not implemented yet!